content/en/docs/instrumentation/java/automatic/agent-config.md

```diff
@@ -36,13 +36,13 @@ The agent can consume configuration from one or more of the following sources
 (ordered from highest to lowest priority):
 
 - system properties
-- environment variables
+- [environment variables](#configuring-with-environment-variables)
 - the [configuration file](#configuration-file)
-- the
-  [`ConfigPropertySource`](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/javaagent-extension-api/src/main/java/io/opentelemetry/javaagent/extension/config/ConfigPropertySource.java)
-  SPI
+- properties provided by the
+  [`AutoConfigurationCustomizer#addPropertiesSupplier()`](https://github.com/open-telemetry/opentelemetry-java/blob/main/sdk-extensions/autoconfigure-spi/src/main/java/io/opentelemetry/sdk/autoconfigure/spi/AutoConfigurationCustomizer.java#L73)
```

Comment:
perhaps should use https://github.com/open-telemetry/opentelemetry-java/blob/f92e02e4caffab0d964c02a32fe305d6d6ba372e/sdk-extensions/autoconfigure-spi/src/main/java/io/opentelemetry/sdk/autoconfigure/spi/AutoConfigurationCustomizer.java#L73 to ensure that the link points to the correct line even if the file is changed

Reply:
Good point, done.